bwrap.xml

```diff
@@ -336,6 +338,10 @@
       <listitem>
         <para>Copy from the file descriptor <arg choice="plain">FD</arg> to
           <arg choice="plain">DEST</arg>.</para>
+        <para>If <arg choice="plain">DEST</arg> already exists, it will be overwritten.
+          Unlike <option>--bind-data</option>, if <arg choice="plain">DEST</arg> or
+          a parent directory is a bind-mount from outside the sandbox,
+          this may overwrite an existing file outside the sandbox.</para>
```

Comment:
I think this part *does* make sense to add, because it clarifies the difference between `--file` and `--bind-data`.